### ISSUE TITLE
For your recipes HTML files, do you need the <html> tags?

### ISSUE THREAD

**Issue by trevor-west** [Jason-B-Jiang/Recipes-Website#2]
Hey Jason, for your recipe HTML files I don't see any <html> tags, or <!DOCTYPE html> tag at the top. I think it would be important for all pages? Or is it fine just to have on your index.html page?

**Comment by Wizzykin** [comment]
> Hey Jason, for your recipe HTML files I don't see any tags, or tag at the top. I think it would be important for all pages? Or is it fine just to have on your index.html page?

He should have the full boilerplate on every page. I noticed that issue too, when I checked out his code.

That template is sort of the minimal data you need to have the skeleton for a website; then you start adding other stuff. Every page is its own thing; the index page is in no way linked to the recipes pages. They all need to have that code to ensure they function properly.

So his code could potentially have issues with browsers not knowing what version is being used, encoding, etc.

**Comment by DakuwoN** [comment]
I had put my boilerplate and it did not link to the other pages, then I looked at the code on this repo, and ended up changing it to <main></main> at the top and bottom, then it worked... ?? Haha! Amazing... I assume the element is telling the browser to reference to the main directory?

**Comment by subraatakumar** [comment]
>Hey Jason, for your recipe HTML files I don't see any tags, or tag at the top. I think it would be important for all pages? Or is it fine just to have on your index.html page?

Yes, all the individual web pages (HTML files) must use HTML boiler plate. It will not create any issue. If you have any issue means some thing is wrong in your code. 

Please [click here](https://github.com/subratsir/odin-recipes) to have a look at my code.

**Comment by ghost** [comment]
we learned this in the foundation🙄one of the very first things learned was the boiler plate must go on every html page.🙄

**Comment by EdwardDevelops** [comment]
love this project!
